scripts/pre_commit_hook.py

```diff
@@ -54,7 +54,7 @@
     'FIREBASE_CONFIG_APP_ID', 'FIREBASE_CONFIG_AUTH_DOMAIN',
     'FIREBASE_CONFIG_DATABASE_URL', 'FIREBASE_CONFIG_MESSAGING_SENDER_ID',
```

Comment:
Do you still need FIREBASE_CONFIG_DATABASE_URL? I thought we removed it.

Reply:
No we don't. Done.